signup/app/app.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule, NO_ERRORS_SCHEMA } from "@angular/core";
+import { NativeScriptModule } from "nativescript-angular/nativescript.module";
+import { AppRoutingModule } from "./app-routing.module";
+import { AppComponent } from "./app.component";
+
+import { SignupComponent } from "./signup/signup.component";
+import { HomeComponent } from "./home/home.component";
+
+import { NativeScriptFacebookModule } from "nativescript-facebook/angular";
+
+import * as application from 'application';
+var nsFacebook = require('nativescript-facebook');
+
+application.on(application.launchEvent, function (args) {
+    nsFacebook.init("1771472059772879");
+});
+
+@NgModule({
+    bootstrap: [
+        AppComponent
+    ],
+    imports: [
+        NativeScriptModule,
+        AppRoutingModule,
+        NativeScriptFacebookModule
```

Comment:
I'm not sure how these page templates will work, but if we just copy the app/signup folder, then it should also have a module and the NativeScriptFacebookModule should be imported there. What do you think?